subsys/usb/device_next/class/Kconfig.cdc_acm

```diff
@@ -21,6 +21,15 @@ config USBD_CDC_ACM_STACK_SIZE
 	help
 	  USB CDC ACM workqueue stack size.
 
+config USBD_CDC_ACM_SHORT_PACKET_TERMINATE_ENABLED
+	bool "USB CDC ACM transmission always end with short packet"
+	default n
+	help
+	  USB CDC ACM driver sends a zero-length-packet to USB host
+	  following the last packet transmitted, if its length is
```

Comment:
There is not "USB CDC ACM driver".

Reply:
Hi @jfischer-no , 
I have modified it according to latest context, and stop calling USB CDC ACM a driver as well: 
https://github.com/zephyrproject-rtos/zephyr/blob/c0d442e9c3ed9f63237d46015179f109993a21b0/subsys/usb/device_next/class/Kconfig.cdc_acm#L24-L33
Please help to review and see whether its all good 😉